tests/test_deregister_task_definition.py

```diff
@@ -0,0 +1,232 @@
+import unittest
+from typing import Literal
+from unittest.mock import Mock, call, patch
+
+import boto3
+
+from actions_helper.commands.deregister_task_definition import deregister_task_definition
+from actions_helper.outputs import CreateTaskDefinitionOutput
+
+
+class DeregisterTaskDefinitionTestCase(unittest.TestCase):
+    @patch.object(boto3, "client")
+    def setUp(self, boto3_client) -> None:
```

Comment:
```suggestion
    def setUp(self, boto3_client):
```
We usually don't specify a `None` return type.

Reply:
Solved in https://github.com/moneymeets/action-ecs-deploy/commit/5ad78696a6a97bace1cd2b4eb5ccc00c2ae35673